README.md

```diff
@@ -114,10 +114,9 @@ Install nose using pip:
 pip install nose
 ```
 
-Run the following commands to run the tests:
+Run the following command to run the unit tests:
```

Comment:
Could be 

    The following command runs unit tests:

Seeing *run* twice in the sentence was stressing me